### PR TITLE
feat(DataTable): expose pageButtonsCount config

### PR DESCRIPTION
**Pull request by julesbarr**

### CODE REVIEW

**Review by dclerici-ssc — COMMENTED** [review]
Hey @julesbarr looks really good. One small comment, would be nice to have an example (story) using that pageButtonsCount prop because is a feature requested by consumer. So, we can refer to it when we communicate

**Comment by julesbarr** [comment]
@dclerici-ssc I added the prop to the story controls so it's configurable now

**Review by dclerici-ssc — APPROVED** [review]
LGTM. Please don't merge. We need to upload this along with the pagination fix.

File: src/components/Datatable/Datatable.tsx [hunk 1]
```diff
@@ -49,6 +49,7 @@ function Datatable<D extends Record<string, unknown>>({
   isBatchModuleEnabled = true,
   controlsConfig = {},
   tableConfig = {},
+  pageButtonsCount,
```
Comment (ajkl2533) [inline src/components/Datatable/Datatable.tsx:52]: This should be hidden inside the `tableConfig` property as written in the ticket. This setting is coupled only with the table itself and there is no need to expose it as a separate property

**Review by ajkl2533 — COMMENTED** [review]
Looks good, we just need to move the property inside `tableConfig`

**Pull request merged**